mitodl/mit-open
# Add an article creation page

# What are the relevant tickets?
https://github.com/mitodl/mit-open/issues/160

# Description (What does it do?)
This PR adds support for creating article content via the frontend UI.

# How can this be tested?
1. Log in as a an admin. *Article creation currently limited to admins.*
2. Ensure  `CKEDITOR_ENVIRONMENT_ID`, `CKEDITOR_SECRET_KEY`, `CKEDITOR_UPLOAD_URL` env vars are set.
3. Visit http://localhost:8063/articles/new to create a new article.

*Note:* There is no article listing page right now, but you can view details at http://localhost:8063/articles/ARTICLE_ID and can view available articles at http://localhost:8063/api/v1/articles
